lib/Config/LocationAwareConfigRepository.php

```diff
@@ -276,8 +277,20 @@ private function invalidateConfigCache(): void
     {
         // invalidate container config cache if debug flag on kernel is set
         $servicesConfig = PIMCORE_PROJECT_ROOT . '/config/services.yaml';
```

Comment:
by looking at the comment above, feels like it was supposed to be like
```suggestion
        $kernel = Pimcore::getKernel();
            if ($kernel->isDebug()) {
                    $servicesConfig = PIMCORE_PROJECT_ROOT . '/config/services.yaml';
```